src/main/java/com/minecrafttas/tasmod/events/server/EventLoadstate.java

```diff
@@ -3,13 +3,27 @@
 import java.io.File;
 
 import com.minecrafttas.common.events.EventBase;
+import com.minecrafttas.tasmod.TASmod;
 import com.minecrafttas.tasmod.events.TASmodEventListener;
+import com.minecrafttas.tasmod.util.LoggerMarkers;
 
+/**
+ * Fired when loading a savestate, before the savestate folder is copied
+ * @author Scribble
+ *
+ */
 public interface EventLoadstate extends EventBase {
 	
+	/**
+	 * Fired when loading a savestate, before the savestate folder is copied
+	 * @param index
+	 * @param target
+	 * @param current
+	 */
```

Comment:
you know you're meant to put something after the @param to tell me what the parameter does